gapic/src/main/com/google/gapid/models/Profile.java

```diff
@@ -289,7 +251,27 @@ public boolean hasSlices() {
     }
 
     public List<PerfNode> getPerfNodes() {
-      return perfNodes;
+      return rootNode.children;
+    }
+
+    public PerfNode getPerfNode(Path.Commands link) {
```

Comment:
Lots of magic numbers in this function, can you explain what the various 1s 2s and 3s do by naming them as local variables or something? Maybe just explain them in comments?

Reply:
Done. Added constants.